sdk/communication/Azure.Communication.Common/src/EntraTokenGuardPolicy.cs

```diff
@@ -46,5 +46,34 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
                 return;
             }
         }
+
+        private (bool CacheOutdated, string EntraToken) IsEntraTokenCacheOutdated(HttpMessage message)
+        {
+            var currentEntraToken = string.Empty;
+            message.Request.Headers.TryGetValue("Authorization", out currentEntraToken);
+            return (string.IsNullOrEmpty(_entraTokenCache) || currentEntraToken != _entraTokenCache, currentEntraToken);
+        }
+        private bool IsAcsTokenCacheOutdated()
```

Comment:
nit: add empty line and it reads better to invert this to the positive phrasing of `IsAcsTokenCacheValid` (negate the bool condition accordingly).